docs/src/components/FancyLink.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components";
+import { Figma, Github, ReactJs, Spectrum, Storybook } from "@icons-pack/react-simple-icons";
+import { ColorPicker, Download, FlightDirect, Google, Tips } from "@kiwicom/orbit-components/icons";
+import { Tile } from "@kiwicom/orbit-components";
+
+import useIsUrlExternal from "../hooks/useIsUrlExternal";
+
+interface FancyLinkProps {
+  download: boolean;
+  icon: string | React.ReactNode;
+  title: string;
+  href: string;
+}
+
+const StyledAnchor = styled.a`
+  margin-top: 12px;
+  display: block;
+`;
+
+const FancyLink = ({ download, href, icon, title }: FancyLinkProps) => {
+  const isExternal = useIsUrlExternal(href);
+
+  let usedIcon = icon || <FlightDirect />;
+
+  // If not passing an icon
+  if (typeof icon === "string") {
```

Comment:
in the interface you have `string | React.Node` i suggest you to left it only as `React.Node` or event better will be to type these string cases as union and have it something like that `Icons | React.Node`.